wire-library/wire-runtime/src/jvmMain/kotlin/com/squareup/wire/internal/FieldBinding.kt

```diff
@@ -59,10 +59,20 @@ class FieldBinding<M : Message<M, B>, B : Message.Builder<M, B>> internal constr
   }
 
   private fun getBuilderMethod(builderType: Class<*>, name: String, type: Class<*>): Method {
+    val errorMessage = "No builder method ${builderType.name}.$name(${type.name})"
     try {
       return builderType.getMethod(name, type)
```

Comment:
When generating a non-null builder method, primitive types would be as unboxed but we would look up for the boxed type, and this would crash.